packages/cdk8s-cli/bin/cmds/init.ts

```diff
@@ -67,13 +68,15 @@ async function determineDeps(version: string, dist?: string): Promise<Deps> {
     'npm_cdk8s': `cdk8s@${ver}`,
     'npm_cdk8s_cli': `cdk8s-cli@${ver}`,
     'pypi_cdk8s': `cdk8s~=${version}`, // no support for pre-release
+    'maven_cdk8s': `cdk8s -Dversion=${version}`,
```

Comment:
I think this is how it should be, but I'm not too familiar with Maven.

Reply:
Isn't this part of the init template work? Revert here.

Reply:
Great point, I'll move this over to that PR